modules/service/src/main/scala/lucuma/odb/graphql/OdbMapping.scala

```diff
@@ -226,13 +230,15 @@ object OdbMapping {
                 ProgramElaborator,
                 SubscriptionElaborator,
                 TargetEnvironmentElaborator,
+                TargetGroupElaborator,
                 QueryElaborator,
               ).combineAll
             )
 
           // Override `defaultRootCursor` to log the GraphQL query. This is optional.
           override def defaultRootCursor(query: Query, tpe: Type, env: Env): F[Result[(Query, Cursor)]] =
-            Logger[F].info("\n\n" + PrettyPrinter.query(query).render(100) + "\n") *> super.defaultRootCursor(query, tpe, env)
+            Logger[F].info("\n\n" + PrettyPrinter.query(query).render(100) + "\n") >> 
```

Comment:
I was having a worrying issue with evaluation order with `*>` here so I switched it to `>>` and will investigate further. Unrelated to anything else happening here.